doc/File Contract Negotiation.md

```diff
@@ -59,37 +59,41 @@ Settings Request
 File Contract Creation
 ----------------------
 
+# A few decisions were made regarding the file contract protocol. The first is
+# that the renter should not sign the file contract until the host has formally
+# accepted the file contract. The second is that the host should be the last
+# one to sign the file contract, as the renter is the party with the strong
+# reputation system.
+#
+# Instead of sending a whole transaction each time, the transaction is sent
+# piecemeal, and only the new parts at each step are sent to the other party.
+# This minimizes the surface area of data for a malicious party to manipulate,
+# which means less verification code, which means less chances of having a bug in
+# the verification code.
+#
+# The renter pays for the siafund fee on the host's collateral and contract
+# fee. If a renter opens a file contract and then never uses it, the host does
+# not lose money. This does put the renter at risk, as they may open up a file
+# contract and then watch the host leave, but the renter is spreading the risk
+# over communications with many hosts, and has a reputation system that will help
+# ensure that the renter is only dealing with upstanding hosts.
```

Comment:
double oh. I guess there's not an easy way to make comments in markdown. I'll have to figure out some other way to get this information across.